target/linux/generic/pending-6.6/620-net_sched-codel-do-not-defer-queue-length-update.patch

```diff
@@ -68,14 +65,12 @@ Link: https://bugzilla.kernel.org/show_bug.cgi?id=109581
  	if (!skb) {
  		/* force a pass through old_flows to prevent starvation */
  		if ((head == &q->new_flows) && !list_empty(&q->old_flows))
-@@ -314,15 +329,6 @@ begin:
+@@ -314,13 +329,6 @@ begin:
  	}
  	qdisc_bstats_update(sch, skb);
  	flow->deficit -= qdisc_pkt_len(skb);
--	/* We cant call qdisc_tree_reduce_backlog() if our qlen is 0,
--	 * or HTB crashes. Defer it for next round.
--	 */
--	if (q->cstats.drop_count && sch->q.qlen) {
+-
+-	if (q->cstats.drop_count) {
```

Comment:
LGTM for this patch. By the way, it will be removed in https://github.com/openwrt/openwrt/pull/18618.